_data/other.yml

```diff
@@ -72,6 +72,14 @@ websites:
       tfa: No
       twitter: Azendoo
 
+    - name: BasisCode Compliance
+      url: http://basiscode.com
+      img: http://basiscode.com/wp-content/uploads/2016/10/BasisCode-Compliance.png
+      tfa: Yes
+      sms: Yes
+      email: Yes
+      phone: Yes
```

Comment:
Is there any public documentation for how two-factor authentication is implemented on the site?